Update known issues (#6560)
cloudflare/cloudflare-docs: content/pages/platform/known-issues.md

- Incremental builds are currently not supported in Cloudflare Pages.

> - Uploading a `/functions` directory through the Direct Upload dashboard UI does not work (refer to [Using Functions in Direct Upload](/pages/platform/direct-upload/#using-functions)).
> 
> - Commits/PRs from forked repositories will not create a preview. Support for this will come in the future.

Reviewer: ```suggestion
- Uploading a `/functions` directory through Direct Upload does not work (refer to [Using Functions in Direct Upload](/pages/platform/direct-upload/#using-functions)).

- Commits/PRs from forked repositories will not create a preview. Support for this will come in the future.
```

Author: This is specific to the dash though. Direct Uploads through Wrangler does work

- Cloudflare's Load Balancer does not work with `*.pages.dev` projects; an `Error 1000: DNS points to prohibited IP` will appear.

> - When adding a custom domain, the domain may get stuck verifying due to being unable to validate a request for an SSL on that hostname. In order for the SSL to validate, ensure Cloudflare Access or a Cloudflare Worker is allowing requests to the validation path: `http://{domain_name}/.well-known/acme-challenge/*`.

Reviewer: ```suggestion
- When adding a custom domain, the domain will not verify if Cloudflare cannot validate a request for an SSL on that hostname. In order for the SSL to validate, ensure Cloudflare Access or a Cloudflare Worker is allowing requests to the validation path: `http://{domain_name}/.well-known/acme-challenge/*`.
```

Reviewer: Should this say '..cannot validate a request for an SSL certificate...'?

Author: ```suggestion
- When adding a custom domain, the domain will not verify if Cloudflare cannot validate a request for an SSL certificate on that hostname. In order for the SSL to validate, ensure Cloudflare Access or a Cloudflare Worker is allowing requests to the validation path: `http://{domain_name}/.well-known/acme-challenge/*`.
```
